src/main/java/vivatech/entity/ElectricFurnaceEntity.java

```diff
@@ -80,8 +85,8 @@ protected boolean canExtractEnergy() {
     @Override
     protected void serverTick() {
         if (canRun()) {
-            cookTime++;
-            energy.extractEnergy(Vivatech.ENERGY, CONSUME_PER_TICK, Simulation.ACTION);
+            cookTime += TIER.getSpeedMultiplier();
```

Comment:
That’s not exactly right thing, because there’s already piece of code that does exactly that below, it the recipe time retrieval

Reply:
Yeah, you're right. That's a more sensible place to put it.